apparmor.d/groups/kde/sddm

```diff
@@ -37,7 +41,7 @@ profile sddm @{exec_path} flags=(attach_disconnected,mediate_deleted) {
   ptrace (trace) peer=@{profile_name},
 
   signal (send) set=(term) peer=kwin_wayland,
-  signal (send) set=(kill, term) peer=startplasma,
+  signal (send) set=(kill, term) peer=startplasma{,-x11},
```

Comment:
There is no `startplasma-X11` profile (startplasma handles both X11 and wayland)